src/towncrier/test/test_build.py

```diff
@@ -1631,6 +1631,29 @@ def test_invalid_fragment_name(self, runner):
         self.assertEqual(1, result.exit_code, result.output)
         self.assertIn("Invalid news fragment name: feature.124", result.output)
 
+    @with_project(
+        config="""
+        [tool.towncrier]
+        package = "foo"
+        template = ["template.jinja"]
```

Comment:
I don't understand why this is not the full path to the template

```suggestion
        template = "foo/newsfragments/template.jinja"
```